.gitignore

```diff
@@ -1,4 +1,147 @@
-#clion build folders.
-cmake-build-*/*
-#clion settings.
-.idea/*
\ No newline at end of file
+#vscode
+.vscode/
+
+# unix
+*.o
+*.so
+!asm/*.o
+
+# temp scripts used for debugging
+*makevs/jdll/*.ijs
```

Comment:
A lot of these ignores are J-specific, which should not be relevant in this repository.

Reply:
I think I got them all :) 

https://github.com/codereport/An-Algorithm-Library/pull/7/commits/6f8abcaa9541c282d88bf2111945087ac15539e3